contracts/factory/src/contract.rs

```diff
@@ -574,6 +576,45 @@ pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, Co
                 };
 
                 CONFIG.save(deps.storage, &new_config)?;
+
+                let keys = migration::PAIR_CONFIGSV110
```

Comment:
it's same as on line 599, so maybe lets move to a separate migrate_pair_configs function

Reply:
Done, thanks.